src/main/java/training/busboard/cli/DataManager.java

```diff
@@ -46,12 +47,18 @@ private void runNextBusesByPostcodeService() {
         String inputPostcode = ui.getPostcode();
         LOGGER.debug("Received post code: " + inputPostcode);
         try {
-            List<Bus> nextBuses = new NextBusesByPostcodeService().getNextBuses(inputPostcode, numBusesToPrint);
+            List<Bus> nextBuses = NextBusesByPostcodeService.getNextBuses(inputPostcode, numBusesToPrint);
             ui.printBuses(nextBuses);
         } catch (Exception e) {
             LOGGER.error("Exception thrown whilst requesting postcode information", e);
             System.out.println("There was a problem getting information about the postcode: " + inputPostcode);
             return;
         }
     }
+
+    public String[] getNextBusesFromPostcodeWeb(String postcode, int numBusesToPrint) {
```

Comment:
It's a bit obtuse to have this return an array of Strings rather than the buses themselves!